src/main.ts

```diff
@@ -82,6 +82,6 @@ import * as process from 'process';
     core.debug(`Done`);
     core.setOutput('version', version);
   } catch (error) {
-    core.setFailed(error.message);
+    core.setFailed((error as Error).message);
```

Comment:
What's this for?

Reply:
@armenzg TS was yelling at me for this error, need to cast as an error or else compilation fails